backend/src/test/java/com/katelocate/menugenerator/RecipeRepositoryTests.java

```diff
@@ -103,7 +103,7 @@ void shouldGetAllRecipes() {
                 new Recipe(0, "Breakfast", RecipeType.BREAKFAST, "[\"2 eggs\", \"salt\", \"peppa\"]", "[\"Scramble eggs\"]"),
                 new Recipe(1, "Dinner", RecipeType.DINNER, "[\"1 tortilla\", \"tomatoes\", \"ground beef\", \"rice\", \"corn\"]", "[\"Burrito\"]"),
                 new Recipe(2, "Snack", RecipeType.SNACK, "[\"3 potatoes\", \"salt\", \"peppa\", \"1 tbsp of cooking oil\"]", "[\"Potato chips\"]"),
-                new Recipe(3, "Dessert", RecipeType.DESSERT, "[\"glutinous rice flour\", \"sugar\", \"water\"]", "[\"Mochi\"]"),
+                new Recipe(3, "Dessert", RecipeType.DESSERT, "[\"glutinous rice flour\", \"sugar\", \"water\"]", "[\"Mochi\"]")
```

Comment:
nit: b734f8e0b2fbb0d73f8045e20ac1223f28b4a4a9 can be fixed up with bd1f0d647a18bebb9153ee066d75a4a49ae0de3d